dts/bindings/arm/arm,dtcm.yaml

```diff
@@ -0,0 +1,17 @@
+
+# SPDX-License-Identifier: Apache-2.0
+title: DTCM
```

Comment:
For reference, we have nxp,imx-dtcm.yaml/nxp,imx-itcm.yaml in the codebase. I understand it wouldn't be used here, so mostly a ping to @MaureenHelm to double-check if iMX's DTCM/ITCM are anyhow special, or it would make sense to use this newly added `arm,dtcm`.